controllers/storagecluster/storageclient.go

```diff
@@ -102,6 +103,9 @@ func (s *storageClient) updateClientConfigMap(r *StorageClusterReconciler, names
 		clientConfig.Data = map[string]string{}
 	}
 	clientConfig.Data[manageNoobaaSubKey] = strconv.FormatBool(false)
+	if !storageCluster.Spec.HostNetwork {
```

Comment:
Do we need a if check? cant we just do `clientConfig.Data[csiControllerPluginHostNetwork] = strconv.FormatBool(storageCluster.Spec.HostNetwork)`?